pydarn/plotting/fan.py

```diff
@@ -27,7 +27,7 @@
 # Third party libraries
 import aacgmv2
 
-from pydarn import PyDARNColormaps, build_scan, radar_fov
+from pydarn import PyDARNColormaps, build_scan, radar_fov, SuperDARNRadars
```

Comment:
```suggestion
from pydarn import PyDARNColormaps, build_scan, radar_fov
```
Imported but not used